content/post/deploy-mssql-server-in-aks/index.md

```diff
@@ -366,6 +381,16 @@ music
 kubedb_system                                                                                                                   
 
 (6 rows affected)
+
```

Comment:
Let's see the database `music` has been created and added to the `availability group cluster`. we will insert data to this `availabily database`